openreview/venue/venue.py

```diff
@@ -80,6 +80,9 @@ def get_id(self):
     def get_short_name(self):
         return self.short_name
     
+    def get_contact(self):
```

Comment:
do you need to implement the get function?

Reply:
I don't think it's necessary. I was just following `short_name` as an example, but it can be removed

Reply:
you can remove it, get_short_name was implemented so it could be polymorph to the conference class. The venue request scripts call this method for both classes.

Reply:
Makes sense, fixed now